### PR TITLE
CE spawns with the Aurora Belt MkII

### PR DESCRIPTION
**Pull request by Cherman0**
## About the PR 

Makes the CE spawn with the aurora belt, removes it from their locker.

## Why's this needed? 

It's somewhat awkward that they dont spawn with their tools. In the force field rework it was changed to reduce melee damage by 75% instead of giving you a 75% chance to dodge all melee attacks, making it significantly less valuable for antags to steal (ie it does nothing against grabs, disarms, batons, etc now). As such I dont feel that it will be an issue that it is not available at all if there's no CE, and it also introduces the benefit that the CE cryoing with their belt doesnt lock it out of existance for any future latejoining CE.

## Changelog 

```changelog
(u)Cherman0
(+)The CE now spawns wearing their Aurora Belt MkII
```

### CODE REVIEW

**Comment by blackrep** [comment]
Doesn't this cause issues with spy thief? The CE's belt can be one of the targets.

**Comment by Cherman0** [comment]
There's already code for removing possible bounties for jobspawn items if there are zero of that job (hos beret and captain auth disk) so i can just add the aurora belt to that

**Comment by Tarmunora** [comment]
note this the sequestered cloner prefab will need updating, as a fully-filled CE locker (read: belt) is the main reward

**Comment by Cherman0** [comment]
Yeah, those lockers are empty. Theyre just generic command lockers, not CE lockers.

File: code/obj/storage/secure_closets.dm [hunk 1]
```diff
@@ -354,7 +353,29 @@ ADMIN_INTERACT_PROCS(/obj/storage/secure/closet, proc/break_open)
 		/obj/item/clothing/suit/space/light/engineer,
 		/obj/item/clothing/head/helmet/space/light/engineer,
 	#endif
+	)
 
+/obj/storage/secure/closet/command/chief_engineer/puzzle
```
Comment (Tarmunora) [inline code/obj/storage/secure_closets.dm:358]: will also need to add it to the prefab

**Review by Tarmunora — APPROVED** [review]

File: code/obj/storage/secure_closets.dm [hunk 2]
```diff
@@ -354,7 +353,29 @@ ADMIN_INTERACT_PROCS(/obj/storage/secure/closet, proc/break_open)
...
+	//obj/storage/secure/closet/command/chief_engineer/puzzle would never be real
+	//obj/storage/secure/closet/command/chief_engineer/puzzle:
```
Comment (ZeWaka) [inline code/obj/storage/secure_closets.dm:360]:
```suggestion
```

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**